src/main/java/net/atos/zac/app/planitems/PlanItemsRESTService.java

```diff
@@ -113,7 +116,9 @@ public void doHumanTaskplanItem(final RESTHumanTaskData humanTaskData) {
         final UUID zaakUUID = caseVariablesService.readZaakUUID(planItem.getCaseInstanceId());
         final Zaak zaak = zrcClientService.readZaak(zaakUUID);
         assertPolicy(policyService.readZaakRechten(zaak).getAanmakenTaak());
-        final HumanTaskParameters humanTaskParameters = zaakafhandelParameterService.findHumanTaskParameters(planItem);
+        final HumanTaskParameters humanTaskParameters =
+                zaakafhandelParameterService.readZaakafhandelParameters(UriUtil.uuidFromURI(zaak.getZaaktype()))
```

Comment:
Zaaktype heeft een convenience method: getUUID() Dan is die UriUtil.uuidFromURI(zaak.getZaaktype()) niet nodig.